packages/uiweb/src/lib/dataProviders/ChatDataProvider.tsx

```diff
@@ -111,28 +72,30 @@ const resetStates = () => {
 useEffect(() => {
     (async () => {
       let user;
-      if (account) {
-        user = await fetchChatProfile({ profileId: account,env });
-        if (user) setConnectedProfile(user);
+      if (!pushUser && signer) {
+        console.log("userrr",user);
+        if (user) {
+          setPushUser(user);  
+        } 
```

Comment:
what does this condition mean

Reply:
https://github.com/ethereum-push-notification-service/push-sdk/pull/852/commits/bff2e193bc20359d0b1f80828378cb06830f9d21#diff-09cd3b29295dd9aa7c4597a09653d6700b9bca9086110130257a85b6cf1e36a5

Follow this for chatDataProvider code

Reply:
Current code is not right